lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -183,7 +188,9 @@ def get_rendered_body(self, obj):
         """
         Returns the rendered body content.
         """
-        return render_body(obj["body"])
+        if self._rendered_body is None:
+            self._rendered_body = render_body(obj["body"])
+        return self._rendered_body
```

Comment:
Can you add details of this change?

Reply:
I will add a comment about this, but I'm essentially caching the rendered_body so that we don't need to render it twice, once for the rendered body itself and once for the preview.

Reply:
Got it. You'd need to take a look at the following decorator function for caching. 
> decorator that automatically handles caching its return value for
    the duration of the request. It returns the cached value for subsequent
    calls to the same function, with the same parameters, within a given request.

https://github.com/edx/edx-platform/blob/8de47ef51f81a8980dad23a7081c56ac0611a0b8/openedx/core/lib/cache_utils.py#L19

Reply:
I think that is an overkill. If we serialise a 20-30 comments in a request, we just need this value to survive for a single serialisation session, not even a request.